pkgs/development/python-modules/pyopenssl/default.nix

```diff
@@ -31,6 +32,12 @@ buildPythonPackage rec {
     "doc"
   ];
 
+  patches = [
+    # fix for oauth2client used by google-cloud-sdk
+    # see: https://github.com/googleapis/google-api-python-client/issues/2554
+    ./revert-1374-Remove-APIs-that-have-been-deprecated-for-a-year.patch
```

Comment:
:-1: Please fix that in google-cloud-sdk using `packageOverrides`.

Reply:
I've added the fix for google-cloud-sdk in a separate commit. The patch won't apply on older versions of pyopenssl so I suggest merging everything together. Thank you for the review!

Reply:
Oppened the separate PR as suggested in commet below.